cob_sound/package.xml

```diff
@@ -26,6 +26,7 @@
   <depend>std_msgs</depend>
   <depend>std_srvs</depend>
   <depend>visualization_msgs</depend>
+  <depend>vlc</depend>
```

Comment:
why do we need `<depend>vlc</depend>` if we have the `<depend>libvlc-dev</depend>` defined?